ocean_provider/validation/RBAC.py

```diff
@@ -42,7 +44,9 @@ def get_action_mapping():
         }
 
     def fails(self):
-        return False
+        payload = self.build_payload()
+        response = requests.post(f"http://{os.getenv('RBAC_SERVER_URL')}", json=payload)
```

Comment:
- the RBAC_SERVER_URL must come already prefixed! do not format it here (do not prepend http://)
- bool(response) will not do what you think it does. It will return True for all responses with a 200 status code, even if their content is "false". Use json.loads(). Have a look in the equivalent PR I made for aquarius.